pxr/imaging/rprUsd/hdMtlxFixed.cpp

```diff
@@ -229,7 +229,15 @@ _AddMaterialXNode(
     // For each of the HdNode parameters add the corresponding parameter/input 
     // to the mxNode
     for (auto const& currParam : hdNode.parameters) {
-        
+
+        // DANGER: Dirty hack
+        // For some reason USD do not provide colorspace attribute on file (or I haven't found it)
```

Comment:
I think there is some work in progress.  Let me investigate this.

Reply:
I updated the description. The "colorSpace" metadata is not supported yet.